src/vulkan/device.h

```diff
@@ -83,9 +83,9 @@ class Device {
   VkInstance instance_ = VK_NULL_HANDLE;
   VkDebugReportCallbackEXT callback_ = VK_NULL_HANDLE;
   VkPhysicalDevice physical_device_ = VK_NULL_HANDLE;
-  VkPhysicalDeviceProperties physical_device_properties_ = {};
-  VkPhysicalDeviceMemoryProperties physical_memory_properties_ = {};
-  VkPhysicalDeviceFeatures available_physical_device_features_ = {};
+  VkPhysicalDeviceProperties physical_device_properties_;
+  VkPhysicalDeviceMemoryProperties physical_memory_properties_;
+  VkPhysicalDeviceFeatures available_physical_device_features_;
```

Comment:
I know this works fine but do you think not initializing these is safe?

Reply:
I think these are fine, if anything comes up we can fix it.